.github/workflows/push-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+    tags:
+      - v*
+jobs:
+  push-master-image-on-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
```

Comment:
Why?

Reply:
docker/setup-qemu action installs QEMU static binaries, which are used to run builders for architectures other than the host.`